core/src/main/res/values/styles.xml

```diff
@@ -296,7 +296,10 @@
     </style>
 
     <style name="AntennaPod.TextView.UnreadIndicator" parent="@android:style/TextAppearance.Small">
-        <item name="android:textSize">@dimen/text_size_micro</item>
+        @dimen/text_size_micro
+        @color/new_indicator_green
+        @string/new_label
+        true
```

Comment:
I have never seen something like this before. Is this correct? I thought those need to be wrapped in `<item>`

Reply:
No, it's note.
Looks like intellij messed up the auto-style merge.